src/backend/distributed/sql/citus--10.2-4--11.0-1.sql

```diff
@@ -14,6 +14,56 @@
 #include "udfs/worker_drop_sequence_dependency/11.0-1.sql"
 #include "udfs/worker_drop_shell_table/11.0-1.sql"
 
+#include "udfs/get_all_active_transactions/11.0-1.sql"
```

Comment:
#include "udfs/get_all_active_transactions/11.0-1.sql"
create view

....

Reply:
Would be nicer to also put citus_lock_waits in a separate file to track changes.

Reply:
Yeah, we considered that, but files are under `udfs` folder and this is a view. So, we weren't sure if it'd make sense to put there.

Ozan will quickly check if we can create `views` folder. If not, we may discuss whether we want to work on adding `views` folder or keep it here.

Reply:
Our current convention, although not ideal, is to create views in the `udfs` file. So I did the same.
And created an issue to fix this: #5691

Reply:
Thanks for acting on this